code/datums/elements/content_barfer.dm

```diff
@@ -0,0 +1,27 @@
+/**
+ * Content Barfer; which expels the contents of a mob when it dies, or is transformed
+ *
+ * Used for morphs and bileworms!
+ */
+/datum/element/content_barfer
+	argument_hash_start_idx = 2
+
+/datum/element/content_barfer/Attach(datum/target, tally_string)
+	. = ..()
+
+	if(!isliving(target))
+		return ELEMENT_INCOMPATIBLE
+
+	RegisterSignals(target, list(COMSIG_LIVING_DEATH, COMSIG_LIVING_ON_WABBAJACKED), PROC_REF(barf_contents))
+
+/datum/element/content_barfer/Detach(datum/target)
+	UnregisterSignal(target, list(COMSIG_LIVING_DEATH, COMSIG_LIVING_ON_WABBAJACKED))
+	return ..()
+
+/datum/element/content_barfer/proc/barf_contents(mob/living/target)
+//	SIGNAL_HANDLER
```

Comment:
```suggestion
	SIGNAL_HANDLER
```

Reply:
doesn't work, rather leave it like this then fix forcemove sometime later